backend/worker/kafka_worker.go

```diff
@@ -453,6 +470,21 @@ func (k *KafkaBatchWorker) flushTraces(ctx context.Context, traceRows []*clickho
 	return nil
 }
 
+func (k *KafkaBatchWorker) flushSessionEvents(ctx context.Context, sessionEventRows []*clickhouse.SessionEventRow) error {
+	span, ctxT := util.StartSpanFromContext(ctx, fmt.Sprintf("worker.kafka.%s.flush.clickhouse", k.Name), util.WithHighlightTracingDisabled(true))
+	span.SetAttribute("NumTraceRows", len(sessionEventRows))
+	span.SetAttribute("PayloadSizeBytes", binary.Size(sessionEventRows))
+	err := k.Worker.PublicResolver.Clickhouse.WriteSessionEventRows(ctxT, sessionEventRows)
```

Comment:
we should change `WriteSessionEventRows` to not use async inserts since this batch will always be large now. that will help reduce load on clickhouse since our worker is ensuring the events are batched. this is what we do for logs / traces which are batched by the kafka worker: https://github.com/highlight/highlight/blob/00589a79d338415b469c533a44f8c4f5589f81d3/backend/clickhouse/traces.go#L186-L210

Reply:
Make sense! I was curious if there was a difference in why errors and sessions should be written asynchronously? Because I believe they are on the same DataSyncQueue.

Reply:
i think we just never switched those over after introducing the datasync queue :thinking: 
@mayberryzane do you remember if there was more context?